docs/guides/nrfconnect_factory_data_configuration.md

```diff
@@ -0,0 +1,293 @@
+- `ISO 8601` format is a [date format](https://www.iso.org/iso-8601-date-and-time-format.html) which represents a date given as YYYY-MM-DD or YYYYMMDD.
+- All certificates stored in factory data are given in [X.509](https://www.itu.int/rec/T-REC-X.509-201910-I/en) format.
+
+<hr>
+<a name="Using own factory data"></a>
+
+## Using own factory data
+
+
+
+<hr>
+<a name="Generating factory data"></a>
+
+## Generating factory data
+
+### Creating factory data JSON file
+
+### Verifying using a JSON schema
+
+Ready JSON file containing factory data can be verified using prepared [JSON Schema](https://github.com/project-chip/connectedhomeip/blob/master/scripts/tools/nrfconnect/nrfconnect_factory_data.schema). JSON Schema is a tool for validating the structure of JSON data. For this purpose, checking the correctness of given factory data is helpful. First, JSON Schema contains a list of mandatory parameters that must be stored in factory data. Missing at least one mandatory parameter will cause a checking error. Then, each parameter is checked according to the given pattern in the second phase of Schema work. The process of validating factory data JSON files is optional, but it can help avoid problems with reading parameters by Matter Device so it is strongly recommended to verify a newly created JSON file.
+
+To check JSON file using JSON Schema manually on Linux machine install *php-json-schema* package:
+```
+$ sudo apt install php-json-schema
+```
+
+And use following command:
+```
+$ validate-json <path_to_JSON_file> <path_to_schema_file>
+```
+
+If in the console there are not any outputs, it means that JSON is correct.
+
+JSON file can be checked automatically by the python script during its generation by given a path to JSON schema file as an additional argument:
+```
+$ python generate_nrfconnect_chip_factory_data.py --schema <path_to_schema>
+```
+
+> Note: This [website](https://json-schema.org/understanding-json-schema/) can help better understand JSON schema. 
+
+### Preparing factory data partition on a device
+
+The factory data partition is an area in a device's persistent storage where a factory data set is stored. This area is configured using the [Partition Manager](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/scripts/partition_manager/partition_manager.html), within which all partitions are declared by writing the `pm_static.yml` file. 
+
+Partitions and their sizes may be different according to different nRF Connect boards. Each nRF Connect board that the example supports contains a `pm_static.yml` file in "*connectedhomeip/examples/< example_name >/nrfconnect/configuration/< pcb name >*" directory. 
+
+To prepare an example that support a factory data, a *pm_static* file has to contain a partition named `factory_data`, and its size should be equal to at least 4 kB and be multiple of one Flash page (for nRF52 and nRF53 boards, a single page is 4 kB size).
```

Comment:
Just a thought... factory_data or maybe  mt_factory_data ?

Reply:
I think it is fine, let's leave it as it is.. I thought about other device's factory data (not related to Matter) - but anyway someone could change it if he wants, use "user" key in our format, or name other factory partition differently.